### PR TITLE
[ETCM-395] faucet unit tests

### PR DESCRIPTION
**Pull request by biandratti**
# Description

Add unit test in class FaucetRpcService.
Fix case when we don't have available the faucetHandler (in the actor system)

# Proposed Solution

When we don't have the faucetHandler available, we receive an InternalError.

### CODE REVIEW

**Review by lemastero — APPROVED** [review]
LGTM 👍

File: src/test/scala/io/iohk/ethereum/faucet/jsonrpc/FaucetRpcServiceSpec.scala [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+      rpcAddress = "",
+      keyStoreDir = "",
+      minRequestInterval = 10.seconds,
+      handlerTimeout = 10.seconds,
+      responseTimeout = 10.seconds,
+      supervisor = mock[SupervisorConfig],
+      shutdownTimeout = 15.seconds
+    )
+
+    val fHandler = TestProbe()
```
Comment (mirkoAlic) [inline src/test/scala/io/iohk/ethereum/faucet/jsonrpc/FaucetRpcServiceSpec.scala:145]:
```suggestion
    val faucetHandler = TestProbe()
```

File: src/test/scala/io/iohk/ethereum/faucet/jsonrpc/FaucetRpcServiceSpec.scala [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    val request: SendFundsRequest = SendFundsRequest(address)
+
+    faucetRpcServiceWithoutFaucetHandler.sendFunds(request).runSyncUnsafe(Duration.Inf) match {
+      case Right(_) => fail()
+      case Left(error) =>
+        error shouldBe JsonRpcError.InternalError
+    }
+  }
+
+  it should "answer timeout when tried to get status but the Faucet Handler is disable" in new TestSetup {
```
Comment (mirkoAlic) [inline src/test/scala/io/iohk/ethereum/faucet/jsonrpc/FaucetRpcServiceSpec.scala:117]:
```suggestion
  it should "answer internal error when tried to get status but the Faucet Handler is disable" in new TestSetup {
```

File: src/test/scala/io/iohk/ethereum/faucet/jsonrpc/FaucetRpcServiceSpec.scala [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+    fHandler.setAutoPilot(simpleAutoPilot { case FaucetHandlerMsg.Status =>
+      StatusResponse(WalletAvailable)
+    })
+    faucetRpcService.status(StatusRequest()).runSyncUnsafe(Duration.Inf) match {
+      case Left(error) => fail(s"failure with error: $error")
+      case Right(response) => response shouldBe FaucetDomain.StatusResponse(WalletAvailable)
+    }
+  }
+
+  it should "answer timeout when tried to send funds but the Faucet Handler is disable" in new TestSetup {
```
Comment (mirkoAlic) [inline src/test/scala/io/iohk/ethereum/faucet/jsonrpc/FaucetRpcServiceSpec.scala:106]:
```suggestion
  it should "answer internal error when tried to send funds but the Faucet Handler is disable" in new TestSetup {
```

**Review by mirkoAlic — APPROVED** [review]

**Pull request merged**